src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RENT = 40;
+  const MIN_DISCOUNT = 20;
+  const MAX_DISCOUNT = 50;
+  const DAYS_FOR_MIN_DISCOUNT = 3;
+  const DAYS_F0R_MAX_DISCOUNT = 7;
+
+  if (days < DAYS_FOR_MIN_DISCOUNT) {
+    return days * DAILY_RENT;
+  }
+
+  if (days < DAYS_F0R_MAX_DISCOUNT) {
+    return days * DAILY_RENT - MIN_DISCOUNT;
+  }
+
+  return days * DAILY_RENT - MAX_DISCOUNT;
```

Comment:
You have same calculation 3 times - try do not repeat your self
```suggestion
  const basicCost = days * DAILY_RENT;
  
  if (days >= DAYS_F0R_MAX_DISCOUNT) {
    return basicCost - MAX_DISCOUNT;
  }

  if (days >= DAYS_FOR_MIN_DISCOUNT) {
    return basicCost - MIN_DISCOUNT;
  }

  return basicCost;
```

Reply:
of course, chack again pls